favourite_tastebook/profile_manager/forms.py

```diff
@@ -0,0 +1,86 @@
+from django import forms
+from django.contrib.auth.models import User
+from profile_manager.models import Profile
+
+MAX_BIO_LEN = 1000
```

Comment:
why is it here? All settings should be in settings or separate settings-like file